### DISCUSSION TITLE
Introduction to the SCI (Read this first!)

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/sci#115] [General]
Watch this 20 min summary first: [Quick Start Video](https://drive.google.com/file/d/1cerWue1oUzIReS2AJgOFSphYG1BlV4zx/view )

Main aspects of the SCI:

1. It's a **score** not a total. 

2. It's focus is **reduction** not neutralisation: You cannot reduce your score by buying Offsets, RECs, you can only reduce your score by emitting less carbon through using less energy, less hardware or using energy more intelligently.

3. **Consequential** not attributional: We don't focus on identifying **who** is responsible for emitting carbon (attributional), if you can perform an action that has the consequence of reducing carbon emissions then you should perform that action (consequential)

4. Bias toward **actions**. Strong bias to incentivise action towards reduction. The three actions that reduce carbon emissions of software are using less energy, using less hardware and using energy more intelligently (carbon awareness). The SCI is designed to encourage those actions.

**Comment by will-iamalpine** [comment]
Can we delete this discussion, since it'll hopefully be merged into the main spec via https://github.com/Green-Software-Foundation/software_carbon_intensity/pull/158 

This doesn't add a lot of value where it resides here, and will likely get lost. If it doesn't pop while reading the spec, i'd propose to make changes there.
